common/web/types/src/kmx/kmx-plus.ts

```diff
@@ -87,13 +87,20 @@ export class Name extends Section {
  * into the string table at finalization.
  */
 export class StrsItem {
+  /** string value */
   readonly value: string;
-  constructor(value: string) {
+  /** char value if this is a single-char placeholder item (CharStrsItem) */
+  readonly char?: number;
+
+  constructor(value: string, char?: number) {
     this.value = value;
+    this.char = char;
   }
```

Comment:
It'd be good to have an assertion here, that if `char` is not `undefined`, that it actually is the first char from `value`.